pyproject.toml

```
@@ -0,0 +1,74 @@
+
+[project]
+name = "compliance-users"
+description = "Carbonplan compliance-users"
+authors = [{name = "CarbonPlan", email = "hello@carbonplan.org"}]
+readme = "README.md"
+requires-python = ">=3.10"
+
+license = {text = "MIT"}
+
+dependencies = [
+"pandas",
+"numpy"
+]
+dynamic = ["version"]
+
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+
+[project.optional-dependencies]
+test = [
+    "pytest",
+    "ruff",
+    "pre-commit",
+    "mypy"
+]
+
+[tool.setuptools_scm]
+fallback_version = "9999"
+
+[tool.setuptools.packages.find]
+exclude = ["docs", "tests", "tests.*", "docs.*"]
+
+[tool.setuptools.package-data]
+datatree = ["py.typed"]
+
+
+[tool.mypy]
+files = "compliance_users/**/*.py"
+show_error_codes = true
+
+
+[tool.ruff]
+# Same as Black.
+line-length = 88
```

```suggestion
line-length = 100
```

to match the previous settings